samples/tfm_integration/psa_crypto/prj.conf

```diff
@@ -11,6 +11,7 @@ CONFIG_LOG_DEFAULT_LEVEL=4
 #CONFIG_PSA_SHELL=y
 
 CONFIG_BUILD_WITH_TFM=y
+CONFIG_TFM_MINIMAL=n
```

Comment:
This commit does not seem right. It makes sdk-zephyr dependent on sdk-nrf, where the `TFM_MINIMAL` option is defined.
Why is this actually needed? I could not find a place where the `TFM_MINIMAL` option is enabled by default for these samples, but probably I'm missing something.

Reply:
TFM_MINIMAL option is enabled by default globally.

Reply:
@joerchan couldn't we disable this globally by default and the enable it in the sdk-nrf samples that require it?

Reply:
Changed TFM_MINIMAL to be a part of the choice symbol TFM_PROFILE_TYPE (as TFM_PROFILE_TYPE_MINIMAL) and having it as default enabled.
Then this can be solved by setting TFM_PROFILE_TYPE_NOT_SET, which is an upstream config.